invenio_rest/views.py

```diff
@@ -80,10 +87,25 @@ def make_response(self, *args, **kwargs):
         :raises :py:class:`werkzeug.exceptions.NotAcceptable`: if no media type
         matches current Accept header.
         """
-        best = request.accept_mimetypes.best_match(self.serializers.keys())
-        if best is None:
-            abort(406)
-        return self.serializers[best](*args, **kwargs)
+        # First search for a local serializer if one has been set
+        if self.method_serializers:
```

Comment:
@jirikuncar This also changes the code behavior. The original code enabled to override the serializer for only some media types. But I suppose replacing completely the default serializers makes more sense as there should not be many use cases, if any, for partial override.